.github/DISCUSSION_TEMPLATE/Ideas.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
```

Comment:
This one can be named as _Idea_ - so it will include _Feature requests_, _Proposals_, _Improvements_ etc.